### PR TITLE
Add link to Echo for button

### PR DESCRIPTION
**Pull request by EinarUeland**

### CODE REVIEW

File: frontend/src/components/MissionOverview/UpcomingMissionView.tsx [hunk 1]
```diff
@@ -151,7 +152,13 @@ export function UpcomingMissionView() {
                             onScheduleButtonPress={onScheduleButtonPress}
                             scheduleButtonDisabled={scheduleButtonDisabled}
                         ></ScheduleMissionDialog>
-                        <Button>Make new mission in Echo</Button>
+                        <Button
+                            onClick={() => {
+                                window.open(echoURL + savedAsset)
+                            }}
+                        >
+                            Create mission
```
Comment (GodVenn) [inline frontend/src/components/MissionOverview/UpcomingMissionView.tsx:160]: Why not specify Echo for create mission?

**Review by GodVenn — COMMENTED** [review]
LGTM, just a question

File: frontend/src/components/MissionOverview/UpcomingMissionView.tsx (hunk 1 above)
Comment (EinarUeland) [reply frontend/src/components/MissionOverview/UpcomingMissionView.tsx:160]: As specified in https://github.com/orgs/equinor/projects/85
@aestene can you answer?

**Review by GodVenn — APPROVED** [review]
LGTM

**Pull request merged**